WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -161,8 +162,7 @@ public async Task<IActionResult> GetTransactionsAsync([FromQuery, Required] IEnu
 			try
 			{
 				var hexes = new Dictionary<uint256, string>();
-				IRPCClient batchingRpc = RpcClient.PrepareBatch();
-				List<Task<Transaction>> tasks = new();
+				List<uint256> missingTxs = new();
```

Comment:
Small nit: How about renaming `missingTxs` to `missingTxIds`?

Reply:
We aren't missing the txids.